src/pages/PMT_PaymentWizard.page

```diff
@@ -27,295 +23,287 @@ Copyright (c) 2008 Evan Callahan, evanc@npowerseattle.org, NPower Seattle, 403 2
                                 <c:UTIL_FormField sObj="{!samplePayment}" sObjType="npe01__OppPayment__c" field="npe01__Payment_Method__c" />
                             </div>
                         </div>
+                    </apex:outputText>
+                    <div class="slds-grid slds-grid_align-center">
+                        <div class="slds-col">
+                            <apex:commandButton action="{!calculate}" value="{!$Label.pmtWizardBtnCalcPayments}" rendered="{!AND(NOT(ISNULL(PaymentsSettings)), PaymentsSettings.npe01__Payments_Enabled__c == true, isScheduleWizard == true, NOT(isClosedLost), OR(haveAmount, removePaidPayments))}" styleClass="slds-button slds-button_neutral slds-m-around_medium" reRender="vfForm"/>
+                        </div>
                     </div>
+                    <!-- Show Payments to be Created -->
+                    <apex:outputText rendered="{!AND(NOT(isClosedLost), payments.size>0, OR(haveAmount, removePaidPayments))}" styleClass="slds-p-around_medium">
+                        <div class="slds-section-title_divider">
+                            {!$Label.pmtWizardToBeCreated}
+                        </div>
+                        <table class="slds-table" >
+                            <thead>
+                                <tr class="slds-text-heading_label">
+                                    <th scope="col" class="slds-cell-wrap">{!$Label.pmtWizardPaymentNumber}</th>
+                                    <apex:repeat value="{!pmtFieldSet}" var="fLbl">
+                                         <th scope="col" class="slds-cell-wrap"><span class="slds-truncate">
+                                            <apex:outputText value="{!fLbl.label}"/>
+                                        </span></th>
+                                    </apex:repeat>
+                                </tr>
+                            </thead>
+                            <apex:repeat value="{!payments}" var="p">
+                                <tr>
+                                    <td>
+                                        <div class="slds-form-element">
+                                            <apex:outputText value="{!p.paymentNumber}"/>
+                                        </div>
+                                    </td>
+                                    <apex:repeat value="{!pmtFieldSet}" var="f"> 
+                                        <td>
+                                            <c:UTIL_FormField field="{!f.fieldPath}"
+                                                              sObj="{!p.OppPayment}"
+                                                              sObjType="npe01__OppPayment__c"
+                                                              showLabel="false"
+                                                              required="!f.DBRequired || f.Required}" />
```

Comment:
The value in the required attribute looks like it might be missing an opening curly bracket.

Reply:
@force2b good catch, fixed, thank you!